### PR TITLE
825 Set network date picker, and its default to now

### LINKED ISSUE #825: In mapLayers hover menu, put current date as default in "TARKKAILUPÄIVÄMÄÄRÄ" input field
Inputfield:
```
3.4.2019    X
```

### PR DESCRIPTION
**Pull request by kronken**
Closes #825 

Updates style of date picker in network selector, and make its default value as now

### CODE REVIEW

File: src/components/map/mapControls/NetworkDateControl.tsx [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+}
+
 const NetworkDateControl = observer(
-    ({ selectDate }: { selectDate: (e: ChangeEvent) => void }) => (
+    (props: INetworkDateControlProps) => (
         <div className={s.networkDateControlView}>
             <label>Tarkkailupäivämäärä</label>
-            <input
-                type='date'
-                className={s.networkDateControlInput}
```
Comment (Divinare) [inline src/components/map/mapControls/NetworkDateControl.tsx]: Remove .networkDateControlInput class also

File: src/components/map/mapControls/MapLayersControl.tsx [hunk 2]
```diff
@@ -58,8 +59,8 @@ class MapLayersControl extends React.Component<
         });
     };
 
-    private selectDate = (e: ChangeEvent<HTMLInputElement>) => {
-        NetworkStore.setSelectedDate(e.target.value);
+    private selectDate = (date: Date) => {
+        NetworkStore.setSelectedDate(date ? Moment(date) : undefined);
```
Comment (Divinare) [inline src/components/map/mapControls/MapLayersControl.tsx]: set as null, not undefined?

File: src/components/map/mapControls/MapLayersControl.tsx [hunk 3]
```diff
@@ -88,7 +89,14 @@ class MapLayersControl extends React.Component<
                         toggleSelectedTransitType={this.toggleTransitType}
                         selectedTransitTypes={NetworkStore.selectedTransitTypes}
                     />
-                    <NetworkDateControl selectDate={this.selectDate} />
+                    <NetworkDateControl
```
Comment (Divinare) [inline src/components/map/mapControls/MapLayersControl.tsx]: Isnt NetworkDateControl a bit overkill? Maybe just use DatePicker straightly?

File: src/components/map/layers/NetworkLayers.tsx [hunk 4]
```diff
@@ -149,7 +148,7 @@ class NetworkLayers extends Component<INetworkLayersProps> {
 
         return (
             !selectedTransitTypes.includes(transitTypeCode) ||
-            this.isDateInRanges(selectedDate, dateRanges) ||
+            !this.isDateInRanges(selectedDate, dateRanges) ||
```
Comment (Divinare) [inline src/components/map/layers/NetworkLayers.tsx]: why was ! added? It didn't work before?

File: src/components/map/mapControls/MapLayersControl.tsx (hunk 3 above)
Comment (Divinare) [reply src/components/map/mapControls/MapLayersControl.tsx]: Why do we btw need DatePicker anyway? Can't we just use InputContainer with type='date'? And then DatePicker.tsx could be removed also

**Comment by kronken** [comment]
Good comments, all comments were taken in consideration and implemented.

**Pull request merged**